gprofiler/utils/node.py

```diff
@@ -0,0 +1,223 @@
+from functools import lru_cache
+import shutil
+import traceback
+from typing import Callable
+from gprofiler.metadata import get_exe_version
+from gprofiler.log import get_logger_adapter
+from granulate_utils.linux.ns import get_proc_root_path, is_same_ns, NsType, run_in_ns
+from granulate_utils.linux.process import is_musl
+
+
+from . import add_permission_dir, pgrep_maps, resource_path, pgrep_exe
+
+import json
```

Comment:
Please run `lint.sh` - it runs `black`, `isort` and other linters. Specifically here I can see that `isort` would reorder things.